packages/iTwinUI-react/src/types/react-table-config.ts

```diff
@@ -100,6 +100,15 @@ declare module 'react-table' {
      * If some rows don't have sub-data, it is recommended to pass an empty array to `subRows` for consistent spacing.
      */
     data: D[];
+    /**
+     * Column's resize mode.
+     *  - `next-column` - when resizing it affects current and the next column,
+     *   e.g. when increasing width of current column, next column's width will decrease.
+     *  - `current-column` - when resizing it affects only the current column,
+     *   e.g. when increasing width of the current column, next column's width remains the same.
+     * @default 'next-column'
+     */
+    resizeMode?: 'next-column' | 'current-column';
```

Comment:
Added new prop here instead of `Table.tsx` so I can avoid typescript errors and still have `resizeMode` in table instance.

Reply:
`resizeMode` already exists in `<Image />`. Maybe `columnResizeMode`?
Or maybe reuse some resizeMode names and have `resizeMode?: 'contain' | 'strech'` ? With contain table keeps its width, with strech - it changes.

Reply:
Renamed